test/file.ts

```diff
@@ -2496,14 +2503,16 @@ describe('File', () => {
   describe('getSignedUrl', () => {
     const CONFIG = {
       action: 'read',
-      expires: Date.now() + 2000,
-    };
+      expires: Date.now() + 2000,  // now + 2 seconds
+    } as {action: string, expires: number, version: string};
```

Comment:
Should be a better interface for us to use here somewhere :)

Reply:
fixed!